pkgs/os-specific/linux/prl-tools/default.nix

```diff
@@ -57,109 +64,63 @@ stdenv.mkDerivation rec {
         cp prl_fs/SharedFolders/Guest/Linux/prl_fs/prl_fs.ko $out/lib/modules/${kernelVersion}/extra
         cp prl_fs_freeze/Snapshot/Guest/Linux/prl_freeze/prl_fs_freeze.ko $out/lib/modules/${kernelVersion}/extra
+        ${if aarch64 then "cp prl_notifier/Installation/lnx/prl_notifier/prl_notifier.ko $out/lib/modules/${kernelVersion}/extra" else ""}
+        cp prl_tg/Toolgate/Guest/Linux/prl_tg/prl_tg.ko $out/lib/modules/${kernelVersion}/extra
       )
     fi
 
     ( # tools
-      cd tools
+      cd tools/tools${if aarch64 then "-arm64" else if x86_64 then "64" else "32"}
       mkdir -p $out/lib
 
       if test -z "$libsOnly"; then
         # install binaries
         for i in bin/* sbin/prl_nettool sbin/prl_snapshot; do
           install -Dm755 $i $out/$i
         done
-        # other binaries
-        for i in xorg.7.1/usr/bin/*; do
-          cp $i $out/bin
-        done
-
-        for i in $out/bin/* $out/sbin/*; do
-          patchelf \
-            --interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-            --set-rpath "$out/lib:$libPath" \
-            $i || true
-        done
 
         mkdir -p $out/bin
-        install -Dm755 ../installer/prlfsmountd.sh $out/sbin/prlfsmountd
+        install -Dm755 ../../tools/prlfsmountd.sh $out/sbin/prlfsmountd
         wrapProgram $out/sbin/prlfsmountd \
           --prefix PATH ':' "$scriptPath"
 
-        for i in lib/*.a; do
-          cp $i $out/lib
-        done
-
-        for i in xorg.7.1/usr/lib/libprl_wmouse_watcher.*; do
+        for i in lib/*.0.0; do
           cp $i $out/lib
         done
 
-        mkdir -p $out/lib/udev/rules.d
-        for i in *.rules; do
-          sed 's,/bin/bash,${stdenv.shell},g' $i > $out/lib/udev/rules.d/$i
-        done
+        mkdir -p $out/share/man/man8
+        install -Dm644 ../mount.prl_fs.8 $out/share/man/man8
 
-        (
-          cd xorg.${xorgVer}
-          # Install the X modules.
-          (
-            cd x-server/modules
-            for i in */*; do
-              install -Dm755 $i $out/lib/xorg/modules/$i
-            done
-          )
-          (
-            cd usr/lib
-            libGLXname=$(echo libglx.so*)
-            install -Dm755 $libGLXname $out/lib/xorg/modules/extensions/$libGLXname
-            ln -s $libGLXname $out/lib/xorg/modules/extensions/libglx.so
-            ln -s $libGLXname $out/lib/xorg/modules/extensions/libglx.so.1
-          )
-        )
+        mkdir -p $out/etc/pm/sleep.d
+        install -Dm644 ../99prltoolsd-hibernate $out/etc/pm/sleep.d
       fi
 
-      for i in xorg.7.1/usr/lib/libGL.*; do
-        cp $i $out/lib
-      done
-
-      cd $out
-      find -name \*.so\* -type f -exec \
-        patchelf --set-rpath "$out/lib:$libPath" {} \;
-
-      cd lib
-      libGLname=$(echo libGL.so*)
-      ln -s $libGLname libGL.so
-      ln -s $libGLname libGL.so.1
+      cd $out/lib
+      ln -s libPrlWl.so.1.0.0 libPrlWl.so.1
+      ${if aarch64 then "" else "
```

Comment:
Usual pattern is `${optionalString aarch64 ''...''}`

Reply:
Just to ensure -- does OpenGL work for AArch64 with this change? It seems strange that we skip creating sover symlinks for one architecture.

Reply:
OpenGL support is provided by [virgl](https://kb.parallels.com/en/128518) starting from Parallels Desktop 17, so that it already works without Parallels Tools.

Reply:
I see, so am I correct that we don't need to create these symlinks at all nowadays, even on amd64? We already use kernel 5.10.

Reply:
Indeed. In this case, I think we should only copy ```lib/libPrl*.0.0``` in the install phase.